.github/workflows/build-images.yml

```diff
@@ -5,8 +5,6 @@ on:
       branches:
         - main
   push:
-      branches:
-        - main
```

Comment:
Why removing this?

Reply:
I removed this, so I could test the GH actions on my fork, on the other branch.

I can restore it, but maybe it's a good idea to not force the branch here.